framework/packages/abstract-interface/src/native/version_control.rs

```diff
@@ -193,6 +193,16 @@ where
 
         Ok(module.reference.unwrap_app()?)
     }
+    /// Does not page over the apps
+    pub fn remove_all_apps(&self) -> Result<(), crate::AbstractInterfaceError> {
+        let modules = self.module_list(None, None, None)?.modules;
```

Comment:
I guess it would be used mainly for tests, but `module_list()` uses `DEFAULT_LIMIT` of 10. We can do multiple queries to load all modules and remove them afterwards. Or let's specify what does "Does not page over the apps" means

Reply:
I just added this because I screwed up the deployment and needed to re-deploy the apps.

Reply:
just removed it